src/Nav.js

```diff
@@ -0,0 +1,2 @@
+import React, { Component } from 'react';
```

Comment:
Again...

Reply:
you can pry that one from my cold dead hands